.tekton/client-server-pull-request.yaml

```diff
@@ -166,14 +144,18 @@ spec:
         value: $(params.git-url)
       - name: revision
         value: $(params.revision)
+      - name: ociStorage
+        value: $(params.output-image).git
+      - name: ociArtifactExpiresAfter
+        value: $(params.image-expires-after)
       runAfter:
       - init
```

Comment:
You need to remove the output workspace for the clone-repository task see: https://github.com/securesign/gitsign/blob/main/.tekton/gitsign-pull-request.yaml#L138

Please check the others :)

Reply:
Hmm I am not really sure what you mean with this point? I looked at your example and compared it with the marked lines and can't see what needs to be changed :)

Reply:
Thanks - removed it now :)